### DISCUSSION TITLE
Revealjs: Is it possible to disable fragments on a single slide?

### DISCUSSION THREAD

**Discussion by m-guggenmos** [quarto-dev/quarto-cli#7509] [Q&A] [answered]
### Description

I have fragments enabled in my revealjs presentation. However, for some slides I'd like to disable fragments manually. Is this possible with Quarto?

**Comment by cderv** [comment, marked as answer]
> I have fragments enabled in my revealjs presentation.

How did you enable fragments ? 

Fragments can be added on a per-content basis  if you need to have them only for some part: https://quarto.org/docs/presentations/revealjs/advanced.html#fragments

You may be refererring to `incremental` feature for List though: https://quarto.org/docs/presentations/revealjs/#incremental-lists
In that case, look at the doc link above for the `.nonincremental` class. 

Tips for next time: 
	Sharing a .qmd example helps know what you are really talking about to answer correctly without much guessing. 

Thank you!

**Comment by m-guggenmos** [comment]
Perfect, thank you, that works! 

Just to note that _ideally_ .nonincremental could also be added to an entire slide

```
# My slide title {.nonincremental}

- Point A
- Point B

Something else

- Point C
- Point D

```

but specifying it for each list does the job as well. And sorry, a MRE would have been useful, and, as I realized, would have clarified the misconception giving rise to my question to begin with.

**Reply by cderv** [reply]
It could be a feature request for Pandoc to add this feature. 

For now, I believe you can add a lua filter to add the div with the class when

````markdown
---
title: XXX
format: 
  revealjs:
    incremental: true
filters:
  - slide-nonincremental.lua
---

# My slide title

- Point A
- Point B

Something else

- Point C
- Point D


# My slide 2 {.nonincremental}

- Point A
- Point B

Something else

- Point C
- Point D

````

````lua
local non_incremental

check_slide = function(h)
  local incremental_ori = PANDOC_WRITER_OPTIONS.incremental
  if h.level == 1 or h.level == 2 then
    if h.classes:includes("nonincremental") then
      _, i = h.classes:find("nonincremental")
      h.classes:remove(i)
      non_incremental = true
      return h
    else
      non_incremental = not incremental_ori
    end
  end
end

wrap_div = function(l)
  if non_incremental then
    return pandoc.Div(l, {class = "nonincremental"}), false
  end
end

return{
  traverse = 'topdown',
  Header = check_slide,
  BulletList = wrap_div
}
````

Just an example that could be improved for sure

**Reply by m-guggenmos** [reply]
Awesome, thank you very much. I will give it a try.
